### PR TITLE
Bang & Olufsen add overlay/announce play_media documentation

### PR DESCRIPTION
**Pull request by mj23000**
## Proposed change

Add documentation for announce in play_media.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/113434 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/bang_olufsen.markdown [hunk 1]
```diff
@@ -52,7 +52,7 @@ Device model:
 
 ### play_media services
 
-The Bang & Olufsen integration supports different playback types in the `media_player.play_media` service: playback from URL, activating a favourite, playback from a local file, playing a radio station, activating a Deezer flow and Deezer playlists, albums, and tracks.
+The Bang & Olufsen integration supports different playback types in the `media_player.play_media` service: playback from URL, activating a favourite, playback from a local file, playing a radio station, activating a Deezer flow and Deezer playlists, albums, and tracks and playing files and TTS as an overlay.
```
Comment (c0ffeeca7) [inline source/_integrations/bang_olufsen.markdown]:
```suggestion
The Bang & Olufsen integration supports different playback types in the `media_player.play_media` service: playback from URL, activating a favorite, playback from a local file, playing a radio station, activating a Deezer flow and Deezer playlists, albums, tracks, and playing files and text-to-speech (TTS) as an overlay.
```

File: source/_integrations/bang_olufsen.markdown [hunk 2]
```diff
@@ -150,6 +150,64 @@ data:
   media_content_id: 1234567890
 ```
 
+##### Overlay
+
+Interrupt currently playing media to play an audio message.
```
Comment (c0ffeeca7) [inline source/_integrations/bang_olufsen.markdown]:
```suggestion
Interrupts currently playing media to play an audio message.
```

File: source/_integrations/bang_olufsen.markdown [hunk 3]
```diff
@@ -150,6 +150,64 @@ data:
...
+
+To use the Bang & Olufsen Cloud TTS, use `overlay_tts` as the `media_content_type` and enter message in the `media_content_id` field.
```
Comment (c0ffeeca7) [inline source/_integrations/bang_olufsen.markdown]:
```suggestion
To use the Bang & Olufsen Cloud TTS, use `overlay_tts` as the `media_content_type` and enter a message into the `media_content_id` field.
```

File: source/_integrations/bang_olufsen.markdown [hunk 4]
```diff
@@ -150,6 +150,64 @@ data:
...
+Bang & Olufsen Cloud TTS messages are limited to 100 unique messages a day and are cached for 24 hours.
+
+Extra keys available:
+
+| Service data attribute    | Optional | Description                                                      |
+| ------------------------- | -------- | ---------------------------------------------------------------- |
+| `overlay_absolute_volume` | yes      | Specify an absolute volume for the overlay.                      |
+| `overlay_offset_volume`   | yes      | Specify a volume offset to be added to the current volume level. |
+| `overlay_tts_language`    | yes      | Specify TTS language.                                            |
```
Comment (c0ffeeca7) [inline source/_integrations/bang_olufsen.markdown]:
```suggestion
| `overlay_tts_language`    | yes      | Specify the language used for text-to-speech.                                            |
```

Comment (c0ffeeca7) [inline source/_integrations/bang_olufsen.markdown]: Would it make sense to mention what type of code is used to specify the language? Is there a default, en-us?

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]
Tiny tweaks

File: source/_integrations/bang_olufsen.markdown [hunk 5]
```diff
@@ -150,6 +150,64 @@ data:
...
+To use the Bang & Olufsen Cloud TTS, use `overlay_tts` as the `media_content_type` and enter a message into the `media_content_id` field.
+Bang & Olufsen Cloud TTS messages are limited to 100 unique messages a day and are cached for 24 hours.
+
+Extra keys available:
+
+| Service data attribute    | Optional | Description                                                      |
+| ------------------------- | -------- | ---------------------------------------------------------------- |
+| `overlay_absolute_volume` | yes      | Specify an absolute volume for the overlay.                      |
+| `overlay_offset_volume`   | yes      | Specify a volume offset to be added to the current volume level. |
+| `overlay_tts_language`    | yes      | Specify the language used for text-to-speech.                                            |
```
Comment (c0ffeeca7) [inline source/_integrations/bang_olufsen.markdown]: Would it make sense to mention what type of code is used here to specify the language? Is there a default, en-us?

File: source/_integrations/bang_olufsen.markdown (hunk 4 above)
Comment (mj23000) [reply source/_integrations/bang_olufsen.markdown]: I've added the default language and the language standard.

File: source/_integrations/bang_olufsen.markdown [hunk 6]
```diff
@@ -159,11 +159,11 @@ Bang & Olufsen Cloud TTS messages are limited to 100 unique messages a day and a
...
-| ------------------------- | -------- | ---------------------------------------------------------------- |
-| `overlay_absolute_volume` | yes      | Specify an absolute volume for the overlay.                      |
-| `overlay_offset_volume`   | yes      | Specify a volume offset to be added to the current volume level. |
-| `overlay_tts_language`    | yes      | Specify the language used for text-to-speech.                                            |
+| Service data attribute    | Optional | Description                                                                                      |
+| ------------------------- | -------- | ------------------------------------------------------------------------------------------------ |
+| `overlay_absolute_volume` | yes      | Specify an absolute volume for the overlay.                                                      |
+| `overlay_offset_volume`   | yes      | Specify a volume offset to be added to the current volume level.                                 |
+| `overlay_tts_language`    | yes      | Specify the language used for text-to-speech. Uses the BCP 47 standard. Default value is "en-us" |
```
Comment (c0ffeeca7) [inline source/_integrations/bang_olufsen.markdown]:
```suggestion
| `overlay_tts_language`    | yes      | Specify the language used for text-to-speech. Uses the BCP 47 standard. Default value is "en-us". |
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @mj23000 :+1:

**Pull request merged**